_i18n/es.yml

```diff
@@ -601,3 +601,146 @@ policies:
+    how-do-i-make-my-password-strong: |-
+      Su contraseña debe tener un mínimo de 8 caracteres. Intente usar varias palabras y espacios para crear una frase que pueda recordar. Las contraseñas más largas con muchas palabras son más seguras que las contraseñas cortas con caracteres especiales.
+
+      Al crear una contraseña, también trate de evitar el uso de jerga, errores ortográficos comunes y palabras deletreadas al revés. Además, evite usar el nombre de su empresa, cónyuge o hijos, incluso su mascota, y cualquier otra información personal como su edad o fecha de nacimiento.
+
+      Si ya ha iniciado sesión y está intentando cambiar su contraseña, email o teléfono, se le pedirá que ingrese su contraseña actual antes de poder realizar los cambios.
+
+      Sin embargo, si repetidamente ingresa una contraseña incorrecta, es muy probable que se deba a que una persona malintencionada se está apoderando de su máquina que dejó desatendida. En ese caso y por su seguridad, lo desconectamos después de tres intentos fallidos.
+    how-does-logingov-protect-my-data: |-
+      Login.gov respeta su privacidad. Nosotros recopilamos datos personales en línea solamente para ayudarlo a ejecutar sus solicitudes con agencias federales asociadas con nosotros. login.gov gestiona partes específicas de la página web de esa agencia para que podamos recopilar datos personales con el fin de proporcionar una plataforma segura y privada para que usted pueda acceder a ciertos servicios del Gobierno. Estamos sujetos a nuestra política de privacidad. Login.gov no alquila, vende o divulga su información personal a empresas privadas, contratistas o vendedores con fines de mercadeo.
+
+      Login.gov tiene enlaces a sitios externos del Gobierno federal para los servicios proporcionados por esas agencias. Esos sitios están sujetos a su propia política de privacidad, que suele estar explicada en la política de privacidad y descargo de responsabilidad legal disponible en su sitio web.
+
+      Las cuentas individuales obtienen dos niveles de seguridad. Requerimos la autenticación de dos factores, así como contraseñas seguras que cumplan con los nuevos requisitos de validación y verificación seguras [Instituto Nacional de Estándares de Tecnología](https://www.nist.gov/){:target="_blank"}. Esto incluye la autenticación de dos factores que requiere que inicie sesión con su contraseña y un código de seguridad que usted recibe en su teléfono. La autenticación de dos factores puede ayudar a proteger su cuenta contra contraseñas comprometidas.
+
+      Su información permanecerá segura. Para obtener más información, lea nuestra [política de seguridad y privacidad]({{site.baseurl}}/policy).
```

Comment:
I know this isn't directly related to this PR, but since you're in there... I think this should just be `[política de seguridad y privacidad](site.baseurl/policy)`